src/Root.js

```diff
@@ -48,9 +48,8 @@ class Root extends Component {
       this.epics[key] = epic;
       this.props.epics.add(epic);
       return true;
-    } else {
-      return false;
     }
+    return false;
```

Comment:
Is this a linting rule that got triggered?

Reply:
Yes https://eslint.org/docs/rules/no-else-return
I have mixed feelings about that rule.

Reply:
Yeah, these days I favor symmetry over efficiency when given a choice

Reply:
As a general policy, we want ESLint to be our servant, not our master. Bondage-and-discipline rules like no-nested-ternary and no-plusplus can go, with no period of mourning required. no-else-return definitely belongs in this category!

Reply:
https://github.com/folio-org/eslint-config-stripes/pull/3